vllm/v1/worker/tpu_model_runner.py

```diff
@@ -787,7 +789,7 @@ def capture_model(self) -> None:
             num_reqs_to_sample = MIN_NUM_SEQS
             dummy_hidden = torch.randn((num_tokens, hsize),
                                        device=device,
-                                       dtype=torch.bfloat16)
+                                       dtype=self._output_dtype)
```

Comment:
Can we directly use the output of the `self.model` call, then we don't need such a dtype argument?

Reply:
either we pay an extra forward or the whole loop moves into dummy_run

Reply:
I was just respecting the function signature, so nothing is returned directly from dummy_run